api/metrics-api-jaxrs/pom.xml

```diff
@@ -24,7 +24,7 @@
   <parent>
     <artifactId>hawkular-metrics-parent</artifactId>
     <groupId>org.hawkular.metrics</groupId>
-    <version>0.3.0-SNAPSHOT</version>
+    <version>0.3.1-SNAPSHOT</version>
```

Comment:
I disagree. If projects depend on SNAPSHOTS it's because they want the latest thing, should it break their code over night. If they want more stability, then let's release 0.3.0 and then bump to 0.3.1-SNASPHOT